implementation/test/BTCETHPriceFeedTest.js

```diff
@@ -1,14 +1,13 @@
-const BTCETHPriceFeed = artifacts.require('BTCETHPriceFeed')
-const MockMedianizer = artifacts.require('MockMedianizer')
 
-import { createSnapshot, restoreSnapshot } from './helpers/snapshot'
-const BN = require('bn.js')
-const chai = require('chai')
-const expect = chai.expect
-const bnChai = require('bn-chai')
-chai.use(bnChai(BN))
+const { createSnapshot, restoreSnapshot } = require('../testHelpers/helpers/snapshot.js')
```

Comment:
> I don't remember why I did this. Proabably mocha was getting confused.

Haha if you don't remember why, let's undo it and see if it's still broken.

Reply:
`json` files are okay in the test directory, but I prefer the file outside the test directory, which I think should only contain test files

Reply:
While I'm not really planning on blocking this PR on it, that's not really how this works in any language, for better or worse. `test` directories are more or less universally for test-*related* files, whether helpers or not. At a high level, consider top-level directories to be “compile units”. You wouldn't really need to compile test helpers without tests or vice versa.

This is also why I hate having to move our test contracts into `contracts/`, but we can cross that bridge later.